packages/mobile/src/hooks/useSelectTierinfo.ts

```diff
@@ -0,0 +1,19 @@
+import { ID } from 'audius-client/src/common/models/Identifiers'
+import { makeGetTierAndVerifiedForUser } from 'audius-client/src/common/store/wallet/utils'
+
+import { useSelectorWeb } from './useSelectorWeb'
+
+const getTierAndVerifiedForUser = makeGetTierAndVerifiedForUser()
+
+/**
+ * Wraps our reselect tier selector in useMemo and useSelector
```

Comment:
This comment might not be accurate because we aren't actually using useMemo

Reply:
oh yeah good call, i moved that out :D